pkg/controller/ttl_manager.go

```diff
@@ -320,6 +321,18 @@ func resolveTtlStatus(ns *coreapi.Namespace, processPods func() (bool, time.Time
 		status.active = active
 		status.logger = status.logger.WithField("active", active)
 		if !lastTransitionTime.IsZero() {
+			// If the namespace has an `active` annotation and
+			// it's after the last transition time, use that instead.
+			if activeTimeString, exists := ns.ObjectMeta.Annotations[activeAnnotation]; exists {
+				activeTime, err := time.Parse(time.RFC3339, activeTimeString)
+				// in case of parsing error, the last transition time will be used anyways.
+				if err == nil {
+					status.logger = status.logger.WithField("active-at", activeTime.Format(time.RFC3339))
+					if activeTime.After(lastTransitionTime) {
+						lastTransitionTime = activeTime
```

Comment:
We have a number of cases to test for `resolveTtlStatus` now:

 - active annotation ignored when namespace is active
 - active annotation parsing error is ignored
 - active annotation is ignored when it's before the last transition time
 - active annotation is used when it's after the last transition time